src/Microsoft.AspNetCore.OData.Authorization/Microsoft.AspNetCore.OData.Authorization.csproj

```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <FrameworkReference Include="Microsoft.AspNetCore.App" />
+    <PackageReference Include="Microsoft.AspNetCore.OData" Version="7.4.1" />
```

Comment:
>Microsoft.AspNetCore.OData" Version="7.4.1" /> [](start = 31, length = 46)

why do we need to rely on this?

Reply:
We use AspNetCore.OData for to get `context.ODataFeature()` and to enable `AddOData().AddAuthorization()`

Reply:
We use the `IODataFeature` to retrieve the `EdmModel` and `Path`.